sql/2021/caching/top_hosts_using_immutable.sql

```diff
@@ -0,0 +1,22 @@
+#standardSQL
+# The top domains to use immutable Cache-Control directive.
+SELECT
+  _TABLE_SUFFIX AS client,
+  NET.HOST(url) AS host,
+  COUNT(DISTINCT pageid) AS pages,
+  COUNT(0) AS requests,
+  SUM(COUNT(DISTINCT pageid)) OVER () AS total_pages,
+  SUM(COUNT(0)) OVER () AS total_requests,
+  COUNT(0) / SUM(COUNT(0)) OVER () AS pct_requests
```

Comment:
Are these `COUNTS` going what you think they are given the `WHERE` clause?

Reply:
LGTM